test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -985,6 +985,17 @@ switch ($x)
             $res.CompletionMatches[0].CompletionText | Should -BeExactly $afterTab
         }
 
+        It "Tab completion UNC path with forward slashes" -Skip:(!$IsWindows) {
+            $homeDrive = $env:HOMEDRIVE.Replace(":", "$")
+            $beforeTab = "//localhost/$homeDrive/wind"
+            # it is expected that tab completion turns forward slashes into backslashes
+            $afterTab = "& '\\localhost\$homeDrive\Windows'"
```

Comment:
`//localhost/C$/wind<tab>` works without changes in this PR, so I think it's covered by a different code path.
The test should be `dir //localhost/<tab>`, and see if it can be tab-completed to `dir \\localhost\ADMIN$`

Reply:
`\\localhost\<tab>` results in two choices: `\\localhost\ADMIN$` and `\\localhost\C$` so I would expect with the PR's code change `//localhost/<tab>` also results in two choices. Doesn't the test logic require completions with only one possible result?

Reply:
You can certainly verify both results, and check `$res.CompletionMatches.Count | Should -Be 2`. But I'm not sure if you will see `\D$` and etc if your computer has more than 1 local disks.

Reply:
Went for `//localhost/admin<tab>` so that it doesn't depend on the list of available drives (how did you know my dev box has more than just C:? 😉).